var/spack/repos/builtin/packages/sentieon-genomics/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os.path
+from spack import *
+
+
+class SentieonGenomics(Package):
+    """Sentieon provides complete solutions for secondary DNA analysis.
+    Our software improves upon BWA, GATK, Mutect, and Mutect2 based pipelines.
+    The Sentieon tools are deployable on any CPU-based computing system.
+    Please set the path to the sentieon server with:
+    export SENTIEON_LICENSE=[FQDN]:[PORT]"""
```

Comment:
Most other packages that work like this have this note added

```suggestion
    export SENTIEON_LICENSE=[FQDN]:[PORT]
    
    Note: A manual download is required.
    Spack will search your current directory for the download file.
    Alternatively, add this file to a mirror so that Spack can find it.
    For instructions on how to set up a mirror, see
    http://spack.readthedocs.io/en/latest/mirrors.html"""
```